README.md

```diff
@@ -31,16 +32,29 @@
 </p>
 </br>
 
-Semgrep is a fast, open-source, static analysis tool that finds bugs and enforces code standards at editor, commit, and CI time. Precise rules look like the code you’re searching; no more traversing abstract syntax trees, wrestling with regexes, or using a painful DSL. Code analysis is performed locally (code is not uploaded) and Semgrep runs on uncompiled code.
+Semgrep is a fast, open-source, static analysis tool for finding bugs and enforcing code standards at editor, commit, and CI time. [Get started →.](#getting-started)
+
+Semgrep runs fully on your computer or build environment: **your code is never sent anywhere**.
+
+Its rules look like the code you already write; no abstract syntax trees, regex wrestling, or painful DSLs. Here's a quick rule for finding Python `print()` statements, run it online in Semgrep's Playground by clicking the image:
 
-The [Semgrep Registry](https://semgrep.dev/explore) has 1,000+ rules written by the Semgrep community covering security, correctness, and performance bugs. No need to DIY unless you want to.
+<p align="center">
+    <a href="https://semgrep.dev/s/ievans:print-to-logger"><img src="https://github.com/returntocorp/semgrep/blob/dlukeomalley/readme-intro/doc/playground-example.png" height="281px" alt="Semgrep rule example for finding Python print() statements"/></a>
```

Comment:
The text in this image looks a bit fuzzy / un-crisp to me. Perhaps due to the sizing?
360px height looked a bit better:

```suggestion
    <a href="https://semgrep.dev/s/ievans:print-to-logger"><img src="https://github.com/returntocorp/semgrep/blob/dlukeomalley/readme-intro/doc/playground-example.png" height="360" alt="Semgrep rule example for finding Python print() statements"/></a>
```